caluma/caluma_workflow/serializers.py

```diff
@@ -240,6 +242,21 @@ def create(self, validated_data):
         )
 
         bulk_create_with_history(work_items, models.WorkItem)
+
+        send_event(
+            events.start_case,
+            sender=self.__class__,
+            case=instance,
+            info=self.context["info"],
+        )
+        for work_item in work_items:  # pragma: no cover
+            send_event(
```

Comment:
Those calls take up a lot of space. I think if we could put them into a method of the serializers (mixin?), we could simplify the calls a lot, leading to less clutter in the process (especially regarding code readability). This could imply removing the `self.__class__` parameter as it could be implied, and if we remove the `info` parameter as noted above, an event would in most cases use a single line instead of six as it is now

Reply:
Done as discussed.